### PR TITLE
Fix differences of XML and images for fillLineGap + show line area boundaries

### PR DESCRIPTION
**Pull request by tairt**
- This one part of the solution for issue #233.

- The combined picture of fillLineGap needs to be re-generated.

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Since the image has been regenerated is this now complete @tairt ?

**Comment by palemieux** [comment]
@nigelmegitt @tairt provided two images, which now need to be combined into one.

**Comment by palemieux** [comment]
Closes #233 and #228

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -1040,7 +1040,8 @@
 
           <figcaption>
             Illustrative rendition of the example immediately above with <code>itts:fillLineGap="true"</code> removed (left) or
-            preserved (right). Blue lines have been added to show the boundaries between line areas.
+            preserved (right). Blue lines have been added to show the <em>before-edge</em> and <em>after-edge</em> of each line
+            area, which are coincident for successive line areas.
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: Maybe reword this for clarity to "Blue lines have been added to show the vertical position of the before-edge and after-edge of adjacent line areas; those edges are coincident. The before-edge of the first line and the after-edge of the last line are omitted." ? Just trying to be absolutely clear about what we are showing or not showing here. Not a big deal for me.

Comment (tairt) [inline imsc1/spec/ttml-ww-profiles.html]: I like the original wording for the first part, but would add as Nigel proposed: "The before-edge of the first line and the after-edge of the last line are omitted."

Comment (palemieux) [inline imsc1/spec/ttml-ww-profiles.html]: > "The before-edge of the first line and the after-edge of the last line are omitted."

The last updated to the PR added these blue lines in the graphics.

Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: There's not enough contrast to see those blue lines at the top and bottom - I could not easily tell if they are there or not. So I would say either change the colours somehow so they are all visible or take them out and add the descriptive text - I would be fine with @tairt 's compromise in https://github.com/w3c/imsc/pull/234#discussion_r119624769

Comment (tairt) [inline imsc1/spec/ttml-ww-profiles.html]: @palemieux Thanks for changing the images and the text, both fine for me. Although the lines at top and bottom are less visible than the others, I could still see them, so it would work for me. If you take  one of Nigels proposals that is also fine with me. Possibly the revised version would already work. I made bottom and top lines a bit thicker.
linegap-revised
